ansible/roles/ssc/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+# SSC Configuration
+ssc_enabled: false
+ssc_version: "0.5.0-alpha.2"
+ssc_image: "sagaxyz/ssc:{{ ssc_version }}"
+ssc_namespace: "sagasrv-ssc"
+ssc_rpc_port: 26657
+ssc_p2p_port: 26656
+ssc_grpc_port: 9090
+ssc_metrics_port: 26660
```

Comment:
We only really need `ssc_version` the others can be hardcoded. I don't see any case where we need to customize it for not exposed pod.